Rakefile

```diff
@@ -111,7 +111,7 @@ namespace :build do
       env_vars = ''
       env_vars += 'MOOC_LOCALIZE=1 ' if CDO.localize_apps
       env_vars += 'MOOC_DIGEST=1 ' unless rack_env?(:development)
-      RakeUtils.system "#{env_vars} grunt"
+      RakeUtils.system "#{env_vars} grunt build"
```

Comment:
do we have a specific reason why to use `rebuild` instead of `build`? Is there an issue with `build` that prevents it from correctly updating the built packages when source contents are changed? Blowing away the `browserify-incremental` compilation-cache on every run seems like a big waste of time that would prevent this task from running most efficiently